rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2020-07-18
+nightly
```

Comment:
This needs to stay at its previous value. We use this to prevent breaking the tool every day

Reply:
done, I have updated the PR